types/index.d.ts

```diff
@@ -5920,6 +5944,29 @@ declare namespace math {
       a2: MathCollection
     ): MathJsChain<T>
 
+    /*************************************************************************
+     * Signal functions
+     ************************************************************************/
+    /**
+     * Compute the transfer function of a zero-pole-gain model.
+     */
+    zpk2tf<T extends number | MathArray | MathArray[]>(
+      this: MathJsChain<T>,
+      z: T,
+      p: T,
+      k?: T
+    ): MathJsChain<MathArray | MathArray[]>
+
+    /**
+     * Calculates the frequency response of a filter given its numerator and denominator coefficients.
+     */
+    freqz<T extends number | MathArray | MathArray[]>(
```

Comment:
Same here, the chained definition needs some changes like explained for `zpk2tf`